src/api/test/test_helper.rb

```diff
@@ -89,7 +89,8 @@ def backend_data
 def inject_build_job(project, package, repo, arch, extrabinary = nil)
   job=IO.popen("find #{backend_data}/jobs/#{arch}/ -name #{project}::#{repo}::#{package}-*")
   jobfile=job.readlines.first
-  return unless jobfile
+  return if project == "BrokenPublishing"
```

Comment:
it was also needed before, but the new way ignores the error only for this single case and we get an error now when it happens in some other situation. Before a missing jobfile was always ignored.